OpenRA.Game/Sound/Sound.cs

```diff
@@ -12,9 +12,11 @@
 using System;
 using System.Collections.Generic;
 using System.IO;
+using System.Linq;
 using OpenRA.FileSystem;
 using OpenRA.GameRules;
 using OpenRA.Primitives;
+using OpenRA.Traits;
```

Comment:
Both of these additional usings look like left-overs from previous approaches and should be redundant + removable now.